qiita_pet/handlers/websocket_handlers.py

```diff
@@ -70,3 +72,26 @@ def on_close(self):
         yield Task(self.toredis.unsubscribe, self.channel)
         self.r_client.delete('%s:messages' % self.channel)
         self.redis.disconnect()
+
+
+class SelectSamplesHandler(WebSocketHandler, BaseHandler):
+    @authenticated
+    def on_message(self, msg):
```

Comment:
Are we requiring docstrings in qiita_pet on tornado built-in functions? So far we haven't, but we are re-writing the style guidelines so should have the discussion.